docker-compose.yaml

```diff
@@ -0,0 +1,53 @@
+version: "3.2"
+
+services:
+  starknet:
+    image: ghcr.io/keep-starknet-strange/madara:v0.1.0-testnet-sharingan-beta.7.1.experimental.4@sha256:baff5a0aa1075d0f84b83ed366f56928b4a6a0c3316762df14e8b1656f784729
+    ports:
+      - "9615:9615"
+      - "9944:9944"
+      - "30333:30333"
+    command:
+      - "--rpc-external"
+      - "--rpc-methods=unsafe"
+      - "--dev"
+    networks:
+      - internal
+
+  kakarot-deployer:
+    image: ghcr.io/kkrt-labs/kakarot/deployer:latest
+    environment:
+      - ACCOUNT_ADDRESS=0x3 
+      - PRIVATE_KEY=0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d 
+      - RPC_URL=http://starknet:9944
+      - EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 
+    volumes:
+      - ./deployments:/app/kakarot/deployments
+    depends_on:
+      - starknet
+    restart: on-failure
+    networks:
+      - internal
+
+  kakarot-rpc:
+    image: ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+    ports:
+      - 3030:3030
+    entrypoint: ["./run.sh", "/usr/bin/tini", "--", "/usr/local/bin/kakarot-rpc"]
+    environment:
+      - KAKAROT_HTTP_RPC_ADDRESS=0.0.0.0:3030 
+      - STARKNET_NETWORK=http://starknet:9944
+      - RUST_LOG=trace
+      - EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 
+    volumes:
+      - ./deployments:/usr/src/app/deployments
+      - ./scripts/run.sh:/usr/src/app/run.sh
```

Comment:
I don't really lmike this `run.sh` for just exporting 2 env variable from the previous run, though I'm not sure how to simply remove it

Reply:
Actually we could do what we did for the devnet version, see https://github.com/kkrt-labs/kakarot/blob/a0b5112004e7ec573a30ac5fc31ec8093f6cb07d/.github/workflows/deployments.yml#L50